test/e2e/helm/kuma_helm_deploy_multi_apps.go

```diff
@@ -68,23 +58,20 @@ metadata:
 			clientPodName, err := PodNameOfApp(cluster, "demo-client", TestNamespace)
 			Expect(err).ToNot(HaveOccurred())
 
-			Eventually(func() (string, error) {
-				_, stderr, err := cluster.ExecWithRetries(TestNamespace, clientPodName, "demo-client",
-					"curl", "-v", "-m", "3", "--fail", "test-server")
-				return stderr, err
-			}, "10s", "1s").Should(ContainSubstring("HTTP/1.1 200 OK"))
+			_, stderr, err := cluster.ExecWithRetries(TestNamespace, clientPodName, "demo-client",
```

Comment:
We also have `client.CollectResponse`, which is more readable IMO and doesn't require checking for `200` but either way

Reply:
https://github.com/kumahq/kuma/issues/4799 created task since it's all over the place